src/components/DirectorySettingsModal/DirectorySettingsSchema.jsx

```diff
@@ -43,14 +45,11 @@ export const DirectorySettingsSchema = (existingTitlesArray = []) =>
               'Title cannot contain any of the following special characters: ~%^*_+-./`;{}[]"<>',
               (value) => !specialCharactersRegexTest.test(value)
             )
-        if (type === "collectionName")
-          return schema
-            .transform((value) =>
-              value.trim().replaceAll(" ", "-").toLowerCase()
-            )
-            .matches(
-              slugifyLowerFalseRegexTest,
-              "Title cannot contain any symbols"
-            )
+        return schema
+          .transform((value) => value.trim().replaceAll(" ", "-").toLowerCase())
+          .matches(
+            slugifyLowerFalseRegexTest,
+            "Title cannot contain any symbols"
+          )
```

Comment:
the problem comes when eslint complains about edge cases not being covered cos it can't infer that there's only 3 types @__@ 

the alternative that i can think of is to include the last conditional and returns a default error case that fails validation like so: 

`schema.test("Invalid case", "This is an invalid type of collection!", () => false)` 

wdyt about this approach

Reply:
That sounds like a good approach! Either that or just leaving a comment directly above this part to explain that it's for `collectionName`